pool/allocator.go

```diff
@@ -19,6 +19,7 @@ type Factory interface {
 	SpawnWorkerWithTimeout(context.Context, *exec.Cmd) (*worker.Process, error)
 	// SpawnWorker creates a new WorkerProcess process based on given command.
 	// Process must not be started.
+	// TODO No usages. Maybe delete?
```

Comment:
You may try to update this interface. Moreover, I think we have to change it and use `...options` as the last arg to easily pass worker options. Since the InitBaseWorker is internal.